isaac_msgs/msg/InspectionStateV1.msg

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2021, United States Government, as represented by the
```

Comment:
This message has not changed at all from the previous versions, a bmr rule would be more appropriate

Reply:
When I froze the `InspectionActionResult` message (derived from `InspectionAction`), it was important to freeze any of its subtypes that were subject to possible future changes at the same time. Otherwise, if the subtype changes, invoking the rewrite rule could cause binary corruption. This is mentioned in [the telemetry maintenance guide](https://nasa.github.io/astrobee/html/maintaining_telemetry.html).

In this case, I froze `InspectionActionResult` as `InspectionV1ActionResult` and therefore also its subtype `InspectionState` as `InspectionStateV1`.